TLM/TLM/Util/Record/TrafficRulesRecord.cs

```diff
@@ -85,8 +85,9 @@ public class TrafficRulesRecord : IRecordable {
                 try {
                     record.Transfer(map);
                 }
-                catch {
+                catch (Exception ex) {
                     Log.Error($"could not transfer {record}");
+                    ex.LogException();
```

Comment:
I assume that stacktrace returned from `Log.Error` is not sufficient, right?

Reply:
I always just ex.ToString() :P

Reply:
correct.